source/guide_mumble.rst

```diff
@@ -34,67 +34,61 @@ All relevant legal information can be found here
   HTTP request sent, awaiting response... 302 Found
-  Location: https://github-production-release-asset-2e65be.s3.amazonaws.com/1413319/a7d2b00a-e4a8-11e6-80bb-d224f64df619?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20181006%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20181006T191435Z&X-Amz-Expires=300&X-Amz-Signature=b2af6f60968f675bb8427af7341716d75a1ee162f822abdd878877514b9205be&X-Amz-SignedHeaders=host&actor_id=0&response-content-disposition=attachment%3B%20filename%3Dmurmur-static_x86-66.6.6.tar.bz2&response-content-type=application%2Foctet-stream [following]
-  --2018-10-06 21:14:35--  https://github-production-release-asset-2e65be.s3.amazonaws.com/1413319/a7d2b00a-e4a8-11e6-80bb-d224f64df619?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20181006%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20181006T191435Z&X-Amz-Expires=300&X-Amz-Signature=b2af6f60968f675bb8427af7341716d75a1ee162f822abdd878877514b9205be&X-Amz-SignedHeaders=host&actor_id=0&response-content-disposition=attachment%3B%20filename%3Dmurmur-static_x86-66.6.6.tar.bz2&response-content-type=application%2Foctet-stream
-  Resolving github-production-release-asset-2e65be.s3.amazonaws.com (github-production-release-asset-2e65be.s3.amazonaws.com)... 52.216.2.32
-  Connecting to github-production-release-asset-2e65be.s3.amazonaws.com (github-production-release-asset-2e65be.s3.amazonaws.com)|52.216.2.32|:443... connected.
-  HTTP request sent, awaiting response... 200 OK
-  Length: 11939824 (11M) [application/octet-stream]
-  Saving to: ‘murmur-static_x86-66.6.6.tar.bz2’
+  Location: ...
+  Saving to: ‘murmur-static.zip’
 
-  100%[======================================================================================================================================>] 11,939,824  7.45MB/s   in 1.5s
+  murmur-static.zip     100%[=============================>]   7,39M  4,47MB/s    in 1,7s
 
-  2018-10-06 21:14:37 (7.45 MB/s) - ‘murmur-static_x86-66.6.6.tar.bz2’ saved [11939824/11939824]
+  2022-12-14 20:01:33 (4,47 MB/s) - ‘murmur-static.zip’ saved [7753035/7753035]
 
-  [isabell@stardust ~]$
+  [isabell@stardust mumble]$
 
 
-extract it and rename it to mumble:
+Extract the zip file:
 
 
 .. code-block:: console
 
-  [isabell@stardust ~]$ tar xvjf murmur-static_x86-66.6.6.tar.bz2
-  murmur-static_x86-66.6.6/
-  murmur-static_x86-66.6.6/murmur.x86
-  murmur-static_x86-66.6.6/LICENSE
-  murmur-static_x86-66.6.6/dbus/
-  murmur-static_x86-66.6.6/dbus/weblist.pl
-  murmur-static_x86-66.6.6/dbus/murmur.pl
-  murmur-static_x86-66.6.6/ice/
-  murmur-static_x86-66.6.6/ice/icedemo.php
-  murmur-static_x86-66.6.6/ice/Murmur.ice
-  murmur-static_x86-66.6.6/ice/weblist.php
-  murmur-static_x86-66.6.6/murmur.ini
-  murmur-static_x86-66.6.6/README
-  [isabell@stardust ~]$ mv murmur-static_x86-66.6.6/ mumble/
-  [isabell@stardust ~]$
+  [isabell@stardust mumble]$ unzip -l murmur-static.zip
```

Comment:
With '-l', unzip lists the contents, but does not extract anything.

Reply:
addressed with https://github.com/Uberspace/lab/pull/1456/commits/91e30b104c6d68bca264412bdae01615bf270c7b